WPFWijkagent/WPFWijkagent/database/SocialMediaImageController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Windows;
+using WijkagentModels;
+
+namespace WijkagentWPF.database
+{
+    public class SocialMediaImageController
+    {
+        private readonly DBContext _dbContext = new DBContext();
+
+        private const string Table = "SocialMediaImage";
+        private const string ColMessageID = "SocialMediaMessageID";
+        private const string ColURL = "URL";
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="messageID"></param>
+        /// <param name="url"></param>
+        /// <returns>number of changed rows</returns>
+        public int SetSocialMediaImage(int messageID, string url)
```

Comment:
waarom niet gewoon een socialmedia image obj dit gaf freek namenlijk bij ons als feedback op de code dit is niet oop

Reply:
In eerste instantie deed ik hetzelfde als op andere controllers dat ie het id zou teruggeven van de db maar dat is nieteens in de db gekomen... Dus... Zal het aanpassen 😬